lib/elixir/src/elixir.erl

```diff
@@ -297,3 +297,11 @@ string_to_quoted(String, StartLine, File, Opts) when is_integer(StartLine), is_b
 
 to_binary(List) when is_list(List) -> elixir_utils:characters_to_binary(List);
 to_binary(Atom) when is_atom(Atom) -> atom_to_binary(Atom, utf8).
+
+handle_parsing_opts(File, Opts) ->
+  put(elixir_parser_file, File),
+  case lists:keyfind(wrap_literals_in_blocks, 1, Opts) of {wrap_literals_in_blocks, WrapLiteralsBool} when is_boolean(WrapLiteralsBool) ->
```

Comment:
The style format of Erlang `case`s we use is:

```erlang
case Expression of
  Pattern1 ->
    ...
  Pattern2 ->
    ...
end
```

Here, I think we can actually refactor to 

```erlang
WrapLiteralsInBlocks =
  case lists:keyfind(wrap_literals_in_blocks, 1, Opts) of
    {wrap_literals_in_blocks, Bool} -> Bool;
    _ -> false
  end,

put(elixir_parser_file, File),
put(wrap_literals_in_blocks, WrapLiteralsInBlocks).
```

I think it reads a bit better. Wdyt?